contract.md

```diff
@@ -12,29 +19,52 @@ like to work on, as part of your project (for example, other things related to D
+
+When we are in class, I also want to be as helpful as possible to my group members which I believe starts with actively listening to what they have to say. I want to be able to help them answer their questions, not just hear and think about them. For me to do this though, I will need to fully understand how their project works and ask questions when I am confused. I hope that my questions will point out areas that may not be clear to others that they want to improve. Additionally, by asking questions, I will understand their project on a deeper level which will hopefully allow me to help them to the best of my ability.
+
 ## Success
 
 _At the end of the semester, what would you be proud to show or tell someone about your
 project?_
 
+At the end of the semester, I will be very proud to show others my project if it is fully working and has replaced the old calendar system that I currently use. This would make me very proud as I chose this project topic to solve an issue in my life and I want to be able to use it when it is complete. Since the scope of my project is intended to help students like myself, I would be particularly pleased if other students also found it cool and wanted to use it too. 
+
 ## Assessment
 
 _Looking over your responses to the previous questions, what would you consider to be an
 "A-" for your project? What would you consider to be an "A"?_
+
+A-: Something that immediately came to mind when considering A- work would be well-documented idea evolution. The odds that I stick to my original project plan without fail seems very slim to me, I can already see myself making unforeseen changes. Whether I come up with a new feature that I believe would make my DSLs more useful or I decide to abandon an idea because I realize it is outside the scope of this project, if these ideas are well documented it will make it clear to see what I got out of this project and how I grew as a coder. Also, I believe that A- level work shows consistent progress and effort regardless of the outcome. If this project takes me far longer than I anticipated and I don't make it near the final result, as long as I chipped away at the task at hand and moved forward, I showed that I did the best that I could.
+
+A: I believe that A level work meets all of the requirements of A- level work but additionally clearly demonstrates that a good deal of time and effort was put into this project. This can be shown in a variety of ways including constant idea generation, solid weekly progress, help sought out during office hours, and anything that demonstrates consistent growth and learning. I think that if I meet all of the milestones that I outlined earlier and worked on the skills/concepts that I wanted to, my project would reach this grade.
```

Comment:
I would be extremely excited to try your DSL out at the end of the semester! I think this project has potential use outside of academic settings, such as creating events for mundane “adult” things like reminders to check your bank account, pay utilities, do taxes, etc.

I think a concrete feature you can consider as taking the project to “A” level is creating that front-end tool. Even though there are references to both DSLs you are making, there is still a lot of work to juggle to design and implement two separate DSLs.